src/Momento.Sdk/Incubating/SimpleCacheClient.cs

```diff
@@ -658,6 +658,23 @@ public async Task<CacheListRemoveAllResponse> ListRemoveAllAsync(string cacheNam
         return await this.dataClient.ListRemoveAllAsync(cacheName, listName, value);
     }
 
+    /// <summary>
+    /// Calculate the length of a list in the cache.
+    ///
+    /// A list that does not exist is interpreted to have length 0.
```

Comment:
Should we return the list is missing response if it does not exist? What do we do for ListFetch if the list does not exist?

Reply:
A list of length 1 will be in the cache. If you pop a value, it will have length zero which also means it is removed from the cache. So length zero for a missing list is consistent with this.

ListFetch has a `Status` property (`HIT`/`MISS`) alongside the payload

Reply:
To be continued!